lib/casmutils/mapping/structure_mapping.cxx

```diff
@@ -7,6 +7,37 @@ namespace casmutils
 {
 namespace mapping
 {
+
+namespace
+{
+double atomic_cost_child(const MappingReport& mapped_result, int Nsites)
+{
+    Nsites = std::max(Nsites, int(1));
+    double atomic_vol = mapped_result.reference_lattice.volume() / double(Nsites) / mapped_result.stretch.determinant();
+    return pow(3. * abs(atomic_vol) / (4. * M_PI), -2. / 3.) *
```

Comment:
Where did this definition come from?

Reply:
This definition came directly from the most updated version of JCT's algorithm for score